### PR TITLE
SONARSCRTC-2

### PR DESCRIPTION
**Pull request by dbmeneses**
https://jira.codehaus.org/browse/SONARSCRTC-2

### CODE REVIEW

**Comment by dbmeneses** [comment]
force pushed to squash all commits into one

File: src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java [hunk 1]
```diff
@@ -65,11 +69,18 @@ private void blame(FileSystem fs, InputFile inputFile, BlameOutput output) {
 
     int exitCode = execute(cl, consumer, stderr);
     if (exitCode != 0) {
-      throw new IllegalStateException("The jazz annotate command [" + cl.toString() + "] failed: " + stderr.getOutput());
+      if(ArrayUtils.contains(UNTRACKED_BLAME_RETURN_CODES, exitCode)) {
```
Comment (henryju) [inline src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java]: It seems your formatter configuration is not correct. Please see https://github.com/SonarSource/sonar-developer-toolset

File: src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java [hunk 2]
```diff
@@ -65,11 +69,18 @@ private void blame(FileSystem fs, InputFile inputFile, BlameOutput output) {
...
+        //skips file silently
```
Comment (henryju) [inline src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java]: Maybe you could add a log in DEBUG level with some details to help supporting users that may wonder why a file is not blamed?

File: src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java (hunk 1 above)
Comment (dbmeneses) [inline src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java]: It should be. What is the problem you see here?

Comment (henryju) [inline src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java]: Missing space between if and first parenthesis.

File: src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java [hunk 3]
```diff
@@ -65,11 +69,18 @@ private void blame(FileSystem fs, InputFile inputFile, BlameOutput output) {
...
+        return;
+      }
+      else {
```
Comment (henryju) [inline src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java]: Another formatting issue that is not automatically fixed by Eclipse. "else" should be on the same line than closing "}".

Comment (SonarTech) [inline src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java]: New MINOR Move this "else" on the same line that the previous closing curly brace. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 new issue:
* 1 minor

Details:
* [Move this "else" on the same line that the previous closing curly brace.](https://github.com/SonarCommunity/sonar-scm-jazzrtc/blob/dfa03234f70c1a1dc551537cfac3120c45582e77/src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java#L76) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)

**Comment by henryju** [comment]
LGTM

**Pull request merged**